src/cli_util.rs

```diff
@@ -900,6 +901,13 @@ impl WorkspaceCommandHelper {
         }
     }
 
+    pub(crate) fn revset_symbol_resolver(&self) -> Box<dyn revset::SymbolResolver + '_> {
+        Box::new(DefaultSymbolResolver::new(
+            self.repo().as_ref(),
+            Some(self.workspace_id()),
+        ))
+    }
```

Comment:
Boxing isn't needed here.

Reply:
I meant the concrete type `-> DefaultSymbolResolver<'_>`, but `impl` should also be fine.

Reply:
I switched it to `DefaultSymbolResolver<'_>`